src/test/java/com/crud/tasks/mapper/TrelloMapperTest.java

```diff
@@ -0,0 +1,144 @@
+package com.crud.tasks.mapper;
+
+import com.crud.tasks.domain.TrelloBoard;
+import com.crud.tasks.domain.TrelloCard;
+import com.crud.tasks.domain.TrelloList;
+import com.crud.tasks.dto.TrelloBoardDto;
+import com.crud.tasks.dto.TrelloCardDto;
+import com.crud.tasks.dto.TrelloListDto;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@ExtendWith(SpringExtension.class)
+class TrelloMapperTest {
+
+    @Autowired
+    TrelloMapper trelloMapper;
+
+    @Test
+    void mapToBoards() {
+        //Given
+        List<TrelloBoardDto> trelloBoardsDto = new ArrayList<>();
+
+        TrelloBoardDto trelloBoardDto1 = new TrelloBoardDto("ID-BOARD-1", "Board1", new ArrayList<>());
+        TrelloListDto trelloListDto1 = new TrelloListDto("ID-LIST-1", "List1", false);
+        TrelloListDto trelloListDto2 = new TrelloListDto("ID-LIST-2", "List2", false);
+        TrelloListDto trelloListDto3 = new TrelloListDto("ID-LIST-3", "List3", false);
+        trelloBoardDto1.getLists().add(trelloListDto1);
+        trelloBoardDto1.getLists().add(trelloListDto2);
+        trelloBoardDto1.getLists().add(trelloListDto3);
+
+        TrelloBoardDto trelloBoardDto2 = new TrelloBoardDto("ID-BOARD-2", "Board2", new ArrayList<>());
+        TrelloListDto trelloListDto4 = new TrelloListDto("ID-LIST-4", "List4", false);
+        TrelloListDto trelloListDto5 = new TrelloListDto("ID-LIST-5", "List5", false);
+        TrelloListDto trelloListDto6 = new TrelloListDto("ID-LIST-6", "List6", false);
+        trelloBoardDto2.getLists().add(trelloListDto4);
+        trelloBoardDto2.getLists().add(trelloListDto5);
+        trelloBoardDto2.getLists().add(trelloListDto6);
+
+        TrelloBoardDto trelloBoardDto3 = new TrelloBoardDto("ID-BOARD-3", "Board3", new ArrayList<>());
+        TrelloListDto trelloListDto7 = new TrelloListDto("ID-LIST-7", "List7", false);
+        TrelloListDto trelloListDto8 = new TrelloListDto("ID-LIST-8", "List8", false);
+        TrelloListDto trelloListDto9 = new TrelloListDto("ID-LIST-9", "List9", false);
+        trelloBoardDto3.getLists().add(trelloListDto7);
+        trelloBoardDto3.getLists().add(trelloListDto8);
+        trelloBoardDto3.getLists().add(trelloListDto9);
+
+        trelloBoardsDto.add(trelloBoardDto1);
+        trelloBoardsDto.add(trelloBoardDto2);
+        trelloBoardsDto.add(trelloBoardDto3);
```

Comment:
wydziel to do metody, np `setupTrelloBoardsDtos()`, podobnie dla testu `mapToBoardsDto` - będzie się lepiej czytać taki test